database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -131,7 +152,254 @@ exports.queries = [
+                      parameters: {
+                        label: "Education Level"
+                        options: ["Secondary", "University"]
+                        layout: "inline"
+                      }
+                    }
+                    {
+                      code: "Q2Secondary"
+                      index: 30
+                      title: "Secondary"
+                      elementTypePluginCode: "longText"
+                      category: QUESTION
+                      parameters: { label: "Please enter details for secondary"}
+                      visibilityCondition: {
+                        operator: "="
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.Q1EducationLevel.text"]
+                          }
+                          "Secondary"
+                        ]
+                      }
+                    }
+                    {
+                      code: "Q3UniversityHistory"
+                      index: 40
+                      title: "Education List"
+                      elementTypePluginCode: "listBuilder"
+                      category: QUESTION
+                      parameters: {
+                        label: "Education history"
+                        createModalButtonText: "Add to education history"
+                        modalText: "## Education history entry \\n\\nPlease enter details for university"
+                        displayType: "card"
+                        inputFields: [
+                          {
+                            code: "LB1"
+                            title: "Name of institution"
+                            elementTypePluginCode: "shortText"
+                            category: QUESTION
+                            parameters: { label: "Name of institution" }
+                            isRequired: true
+                          }
+                          {
+                            code: "LB2"
+                            title: "University year"
+                            elementTypePluginCode: "shortText"
+                            category: QUESTION
+                            parameters: {
+                              label: "Year of conclusion"
+                              maxWidth: 150
+                            }
+                            validation: {
+                              operator: "REGEX"
+                              children: [
+                                {
+                                  operator: "objectProperties"
+                                  children: ["responses.thisResponse"]
+                                }
+                                "^(?:(?:18|19|20|21)[0-9]{2})$"
+                              ]
+                            }
+                            validationMessage: "Year between 1900-2100"
```

Comment:
What do you have against people who graduated more than 120 years ago? Ageist!! 😠

Reply:
Hahaha But all good if you came back from the future : P